### PR TITLE
Pico Plush but it actually works this time

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Same as the old one but made from a branch that hopefully isn't borked this time.
Adds a plush of Tom Fulp's character Pico, from Pico's School.
Screenshot 2021-06-07 222736
https://user-images.githubusercontent.com/80640114/120684363-e2919280-c49e-11eb-92c7-109dc06494ad.mp4

## Why It's Good For The Game

It was about dang time we honoured our origins and added NG references. Might as well start with the man we got our name from.

## Changelog
:cl:
 add: Added the Pico plushie
/:cl:

### CODE REVIEW

**Comment by FernandoJ8** [comment]
well isn't that ironic

**Comment by JohnFulpWillard** [comment]
have you considered getting real

**Comment by FernandoJ8** [comment]
I think that's the first time linters succeed. This looks promising

**Comment by Nobelium-XVIII** [comment]
Ah, the man who stopped a school shooting with an unloaded gun

**Comment by EgorDinamit** [comment]
> Ah, the man who stopped a school shooting with an unloaded gun

Wait, it was unloaded?

**Comment by EgorDinamit** [comment]
> why is there like 50 shades for a single sprite lol

image

File: fulp_modules/Z_edits/container_init/machine_init.dm [hunk 1]
```diff
@@ -82,7 +82,8 @@
 	. = ..()
 
 /obj/machinery/vending/games/Initialize()
-	products += list(/obj/item/toy/plush/batong = 3) //Used in toys.dm
+	products += list(/obj/item/toy/plush/batong = 3,
```
Comment (Fikou) [inline fulp_modules/Z_edits/container_init/machine_init.dm:85]: change this list to
list(
    thing 1,
    thing 2
)

File: fulp_modules/features/toys/code/toys.dm [hunk 2]
```diff
@@ -24,3 +24,13 @@
 	attack_verb_continuous = list("dust")
 	attack_verb_simple = list("dust")
 	squeak_override = list('sound/effects/supermatter.ogg'= 1)
+
+/obj/item/toy/plush/pico
+	name = "pico plushie"
+	desc = "A plushie of an alarmed middle schooler with orange hair and a green T-Shirt. Maybe something's happening at his school."
```
Comment (Fikou) [inline fulp_modules/features/toys/code/toys.dm:30]: it should be T-shirt or t-shirt

Comment (EgorDinamit) [reply fulp_modules/features/toys/code/toys.dm:30]: Fun fact: It's not a **T** shirt. It's just a short, look at the sleeves.

File: fulp_modules/Z_edits/container_init/machine_init.dm (hunk 1 above)
Comment (Fikou) [reply fulp_modules/Z_edits/container_init/machine_init.dm:85]: its easier read

**Comment by FernandoJ8** [comment]
>
> why is there like 50 shades for a single sprite lol
>

I mainly used the lizard plushie, or lord and saviour, as reference, which has a lot of shades as well. I think it looks mostly okay the way it is.

**Comment by EgorDinamit** [comment]
Arms, hands specifically, look quite small if you'd ask me.
The mouth as well, it looks like one tooth, but I don't think anyone can make it look good enough in 32x32, so it's just fine.

**Comment by FernandoJ8** [comment]
> 
> 
> Arms, hands specifically, look quite small if you'd ask me.
> The mouth as well, it looks like one tooth, but I don't think anyone can make it look good enough in 32x32, so it's just fine.

The tooth is intentional, I was initially just making an empty mouth but messing around with the possibility of adding teeth I liked the derpy one tooth and decided to keep it. The arms are a problem but yeah, hard to correct in 32x32. Will probably need a remaster at some point but it will do for now, I think it's *just* okay enough

**Pull request merged**